app/components/Blog/BlogPost.tsx

```diff
@@ -39,7 +39,7 @@ export const BlogPost = ({
         flexFlow="column"
         justifyContent="center"
         mt={{ base: "-8rem", md: "-8rem", lg: "-20rem" }}
-        px={{ base: ".5rem", md: "8rem", lg: "12rem", xl: "32rem" }}
+        px={{ base: ".5rem", md: "8rem", lg: "12rem" }}
```

Comment:
And the issue with removing the `xl` breakpoint from here now makes the content too "wide"-ish on screens with the `xl` breakpoint. See 👇🏼 

![image](https://github.com/acekyd/made-in-nigeria/assets/48127194/d7c51344-8c18-4f9c-bb4b-4cdda17cb45c)

Folks would have to be struggling to read content from left to right. It is better if the content is in the center and it catches their attention immediately. 

A `xl` value of `22rem` should suffice, perhaps. In your next PR. But, feel free to play with the values until you get one that is suitable.

Reply:
can't we just add a max-width to the content?